guides/release/accessibility/application-considerations.md

```diff
@@ -87,5 +87,6 @@ Here are some examples of accessibility-focused addons created by many people th
 - [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) - A suite of accessibility tests that can be run within the Ember testing framework
 - [a11y-announcer](https://github.com/ember-a11y/a11y-announcer) - An accessible ember route change announcer
 - [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) - linter for Ember templates
+- [ember-accessibility](https://github.com/coyote-labs/ember-accessibility) - An Ember.js addon to find and address accessibility violations during development. 
```

Comment:
please update this description to indicate that it is the same as ember-a11y-testing but with an in-browser interactive interface. Thank you!

Reply:
I have changed the wording, thanks for your time.